recipes/qcbor/all/conanfile.py

```diff
@@ -18,10 +19,12 @@ class QCBORConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
+        "disable_float": [False, "HW_USE", "PREFERRED", "ALL"],
```

Comment:
Options should be lowercase 🤔 I dont recall if that meant values too

Reply:
It should be applied to values too, because on case-sensitive systems like Linux, it will be affected. We didn't enforce it because `fPIC` is an exception (I would prefer using `fpic` on Conan 2.0 but it would affect all recipes ...)